module-system/sov-default-stf/README.md

```diff
@@ -30,8 +29,7 @@ where
 
 1. The `DispatchCall` trait is responsible for decoding serialized messages and forwarding them to the appropriate module.
 1. The `Genesis` trait handles the initialization process of the rollup. It sets up the initial state upon the rollup deployment.
-1. The `TxVerifier` trait is responsible for validating transactions within the rollup. It ensures that incoming transactions meet the necessary criteria and are valid for execution.
-1. The `TxHooks` trait allows for the injection of custom logic into the transaction processing pipeline. It provides a mechanism to execute additional actions or perform specific operations during the transaction processing phase.
+1. The `TxHooks & ApplyBlobHooks` traits that allows for the injection of custom logic into the transaction processing pipeline. It provides a mechanism to execute additional actions or perform specific operations during the transaction processing phase.
```

Comment:
nit: Since there are two traits, we need to make this plural: 

`The `TxHooks & ApplyBlobHooks` traits that allow for the injection of custom logic into the transaction processing pipeline. They provide a mechanism to execute additional actions or perform specific operations during the transaction processing phase.`

Reply:
Done